File: Code/FizzBuzz.py
# FizzBuzz in Python3   https://github.com/Jasper-27

fizz = 3 
buzz = 5 

for i in range (1, 100): 
    out = ""

    if i % fizz == 0: 
        out += "Fizz"
    if i % buzz == 0: 
        out += "Buzz"
        
    if out == "": 
        print(i)
    else: 
        print(out)

File: Code/FizzBuzz.cs
using System;
					
public class Program
{
	public static int Fizz = 3, Buzz = 5, X = 100;
	
	private static string fizzStr = "Fizz", buzzStr = "Buzz";
	
	private static void FizzBuzz(int fizz, int buzz, int limit) 
	{
		for (int i = 1; i < limit; i++)
		{
			if (i % fizz == 0)
			{
				if (i % buzz == 0)
				{
					Console.Write(fizzStr + buzzStr + " ");
				}
				else 
				{
					Console.Write(fizzStr + " ");
				}
			}
			else if (i % buzz == 0)
			{
				Console.Write(buzzStr + " ");
			}
			else 
			{
				Console.Write(i + " ");
			}
		}
		
		Console.WriteLine();
	}
	
	public static void Main()
	{
		FizzBuzz(Fizz, Buzz, X);
	}
}
